netbox/utilities/templatetags/helpers.py

```diff
@@ -100,7 +100,7 @@ def humanize_megabytes(mb):
         return f'{int(mb / 1048576)} TB'
     if not mb % 1024:
         return f'{int(mb / 1024)} GB'
-    return f'{mb} MB'
+    return f'{int(mb) / (1048576):.2f} MB'
```

Comment:
Additionally, because we're now rounding to the nearest hundredth of a TB or GB, we also need to also convey the exact value in megabytes. This can be achieved by wrapping the value in a `<span>` element with a `title` attribute indicating the raw value, which will be displayed on mouseover.